Sources/MongoSwift/ChangeStreamOptions.swift

```diff
@@ -0,0 +1,101 @@
+/// Describes the modes for configuring the fullDocument field of a
+/// `ChangeStreamDocument`.
+public enum FullDocument: RawRepresentable, Codable {
+    /// The change stream document will include both a delta describing the
+    /// changes to the document, as well as a copy of the entire document that
+    /// was changed from some time after the change occurred.
+    case updateLookup
+    /// For an unknown value. For forwards compatibility, no error will be
+    /// thrown when an unknown value is provided.
+    case other(String)
+
+    public var rawValue: String {
+        switch self {
+        case .updateLookup:
+            return "updateLookup"
+        case .other(let v):
+            return v
+        }
+    }
+
+    public init?(rawValue: String) {
+        switch rawValue {
+        case "updateLookup":
+            self = .updateLookup
+        default:
+            self = .other(rawValue)
+        }
+    }
+}
+
+/// Options to use when creating a `ChangeStream`.
+public struct ChangeStreamOptions: Codable {
+    /** Indicates how the `fullDocument` field of a `ChangeStreamDocument` should
+     * be filled out by the server.
+     * By default (indicated by a nil value for this option), the fullDocument field
+     * in the change stream document will always be present in the case of 'insert'
+     * and 'replace' operations (containing the document being inserted) and will be
+     * nil for all other operations.
+     */
+    public let fullDocument: FullDocument?
+
+    /// Specifies the logical starting point for the new change stream.
+    public let resumeAfter: Document?
```

Comment:
the type here should be a `ChangeStreamToken?`. Also ditto the documentation stuff I said elsewhere about `ChangeStreamToken`s.

Since this type is by nature opaque (and complicated), we need to provide a lot of docs so users can figure out how to use it.